packages/graph/hash_graph/lib/graph/src/store/postgres/mod.rs

```diff
@@ -195,83 +188,47 @@ impl DependencyContext {
 
     #[must_use]
     pub fn into_subgraph(self, roots: HashSet<GraphElementEditionId>) -> Subgraph {
-        let ontology_vertices = OntologyVertices(
-            self.referenced_data_types
-                .into_values()
-                .map(|data_type| {
-                    (
-                        data_type.metadata().edition_id().base_id().clone(),
-                        (
-                            data_type.metadata().edition_id().version(),
-                            OntologyVertex::DataType(Box::new(data_type)),
-                        ),
-                    )
-                })
-                .chain(
-                    self.referenced_property_types
-                        .into_values()
-                        .map(|property_type| {
-                            (
-                                property_type.metadata().edition_id().base_id().clone(),
-                                (
-                                    property_type.metadata().edition_id().version(),
-                                    OntologyVertex::PropertyType(Box::new(property_type)),
-                                ),
-                            )
-                        }),
+        let ontology_vertices = self
+            .referenced_data_types
+            .into_values()
+            .map(|data_type| {
+                (
+                    data_type.metadata().edition_id().clone(),
+                    OntologyVertex::DataType(Box::new(data_type)),
```

Comment:
I didn't see the other PR (maybe I wasn't requested), have we talked about the impact of boxing here? I appreciate the lint was suggesting the enums might be too big but I don't remember discussion around if we wanted to actually follow the lint in this case

Reply:
@thehabbos007 and I spoke about that a while ago. We both agreed that the size difference of this struct is so incredibly big, that it's worth boxing it.

Reply:
```
OntologyVertex: 408
DataTypeWithMetadata: 232
PropertyTypeWithMetadata: 184
EntityTypeWithMetadata: 400
```
were the struct sizes

Reply:
What is the actual impact of boxing though? I agree the structs are quite large but that's not the actual explanation of why boxing is beneficial

Reply:
A `Box` always has the exact same size: `8` for `Sized` and `16` for DSTs, so the enum will be much smaller in size.